CHANGELOG.md

```diff
@@ -59,22 +59,18 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 * You will now need to invoke `allow_tables_to_appear_in_same_query!` any time two tables
   appear together in the same query, even if there is a `joinable!` invocationfor those tables.
 
-* The story about the custom derive annotation changed. All pure custom
- derive procedural macro implementations (`Queryable`, `QueryableByName`,
- `Identifiable`, `Insertable`, `AsChangeset` and `Associated`) are now
- provided by `diesel_derives`. Furthermore those annotations are reexported
- by `diesel` itself, so there is no need to depend on a second crate for
- custom derives anymore. The `infer_schema!` and `infer_table_from_schema!`
- macro is moved from diesel into the `diesel_infer_schema` crate. All
- functionality related to migrations (The `diesel::migrations` module and the
- `embed_migrations!` macro) is moved to the `diesel_migrations` crate. If you
- are using any of those, simply depend on the new crates and your code will
- continue to build.
- This means the following:
-	* You are using `infer_schema!` or `infer_table_from_schema!` -> depend now on `diesel_infer_schema`
-	* You are using `embed_migrations!` or any other migration releated stuff -> depend now on `diesel_migrations`
-	* You are using custom derives from `diesel_codegen` -> remove that dependency and add `#[macro_use]` to `extern crate diesel`
+* `diesel_codegen` should no longer explicitly be used as a dependency. Unless
```

Comment:
`diesel_codegen` not longer exists, it is "renamed" in `diesel_derives`.
So maybe 
> `diesel_codegen` exists no longer and must be removed as dependency. All custom derives are provides by diesel itself now.